### ISSUE TITLE
Document the new choice settings

### ISSUE THREAD

**Issue by kiudee** [kiudee/cs-ranking#33]
Todo
----

- [x] Update the README.rst
- [x] Mirror the intro.rst
- [ ] Update API:
  - [ ] Check old API reference and fix if necessary
  - [ ] Include new settings
- [ ] Write example notebooks for both settings

**Comment by prithagupta** [comment]
- [ ] Install CS-Rank
- [ ] Quick Start
- [ ] Tutorials
	- [ ] Settings
		- [ ] Object Ranking
		- [ ] Discrete Choice
		- [ ] General Choice Functions
	- [ ] Hyperparameter Optimization
	- [ ] Datasets
	(* Extend CS-Rank)
API Reference
	- [ ] Object Ranking
	- [ ] Discrete Choice
	- [ ] General Choice Functions
	- [ ] Loss Functions
	- [ ] Metrics
	- [ ] Hyperparameter Optimization
	- [ ] Dataset Parsers

**Comment by kiudee** [comment]
I will close this for now. It is better to create a new issue in order to work on specific areas.
